src/styles/Elements/buttons.scss

```diff
@@ -0,0 +1,33 @@
+.btn, .form-btn {
+  text-transform: uppercase;
+}
+
+.btn {
+  background: $yellow-500;
+  border-radius: 18px;
+  box-shadow: 1px 1px 2px $black-400;
+  display: inline-block;
+  padding: 7px 13px;
```

Comment:
Use `rem` units everywhere except for `border-radius`, `border`, `box-shadow` :)
Also add variation for `large` `small` `medium` buttons aka different paddings ;)

P.S `1rem` = `10px`